content/reference/merchant-integrations.md

```diff
@@ -0,0 +1,2 @@
+<!-- FIXME: Create a Merchant Integration landing page -->
+<meta http-equiv="refresh" content="0; URL=/guides/merchant-payment-conditions">
```

Comment:
This allows us to have the correct structure in our navigation and do a redirect if `Merchant Integrations` is clicked as we don't have a landing page.

Open to suggestions on how to do this, if we are planning to create a landing page I think this is ok as a placeholder.

Reply:
This is ok for now - there's legacy merchant terminals documentation that we should adapt for this page. I've added a task for this